setup.py

```diff
@@ -34,7 +34,7 @@
         ),
         packages=find_packages(),
         package_data={
-            "orangecontrib.lookalike.widgets": ["icons/*.svg"],
+            "orangecontrib.lookalike.widgets": ["icons/*.svg","data/*.xml"],
```

Comment:
Would you mind adding space between list item? And then squash the commits? Thank you!